### PR TITLE
Update anatomy images

### PR DESCRIPTION
**Pull request by melaniebmn**
# Summary | Résumé

Update anatomy sections to new styling with a number list.

### CODE REVIEW

File: src/fr/composants/pied-de-page/design.md [hunk 1]
```diff
@@ -8,17 +8,21 @@ date: "git Last Modified"
...
-
-Le **mot-symbole Canada** est l'identificateur mondial du gouvernement du Canada et l'un des trois symboles officiels de l'identité du gouvernement du Canada.
+<ol class="anatomy-list">
+  <li>La <strong>bande de liens du pied de page</strong> GC s'étend sur le bas de la page. Elle contient le mot-symbole officiel du gouvernement du Canada et des liens vers les conditions générales et les avis de confidentialité des produits, ainsi que des liens facultatifs vers les pages Canada.ca pour les médias sociaux, les applications, ainsi qu'une section « à propos ».</li>
+  <li>Le <strong>mot-symbole Canada</strong> est l'identificateur mondial du gouvernement du Canada et l'un des trois symboles officiels de l'identité du gouvernement du Canada.</li>
+</ol>
 
 <img class="b-sm b-default p-400" src="/images/fr/components/anatomy/gcds-footer-anatomy-recommended.svg" alt="Website footer taxonomy with two sections. The first section is the Footer links bands in a light grey banner with some links like Social media, Mobile applications, About Canada.ca and so on. The second section is the Canada workmark indicating Canada logo." />
```
Comment (ethanWallace) [inline src/fr/composants/pied-de-page/design.md:16]: I know this isn't related to the work you did in this PR, but just noticing that this `img` has English alt text on the French design page.

Comment (melaniebmn) [reply src/fr/composants/pied-de-page/design.md:16]: Good catch! Asked translation to translate it and they will send it back tomorrow.

File: src/images/en/components/anatomy/gcds-checkbox-anatomy-with-fieldset.svg
Comment (ethanWallace) [inline src/images/en/components/anatomy/gcds-checkbox-anatomy-with-fieldset.svg:1]: The fieldset legend size does not match the new size shown in the Fieldset anatomy image

File: src/images/en/components/anatomy/gcds-radio-anatomy.svg
Comment (ethanWallace) [inline src/images/en/components/anatomy/gcds-radio-anatomy.svg:1]: The fieldset legend size does not match the new size shown in the Fieldset anatomy image

File: src/images/fr/components/anatomy/gcds-checkbox-anatomy-with-fieldset.svg
Comment (ethanWallace) [inline src/images/fr/components/anatomy/gcds-checkbox-anatomy-with-fieldset.svg:1]: The fieldset legend size does not match the new size shown in the Fieldset anatomy image

File: src/images/fr/components/anatomy/gcds-radio-anatomy.svg
Comment (ethanWallace) [inline src/images/fr/components/anatomy/gcds-radio-anatomy.svg:1]: The fieldset legend size does not match the new size shown in the Fieldset anatomy image

**Review by ethanWallace — CHANGES_REQUESTED** [review]
The content and layout looks good!

Just some small tweaks need to be made to some radio and checkbox anatomy images to be consistent .

**Review by ethanWallace — APPROVED** [review]
LGTM

**Pull request merged**